### PR TITLE
Added shareable link functionality

### LINKED ISSUE #91: Add a button to each section to copy link to the section
**New Feature**
The Interactive-Book comprises of various sections and adding a button to copy a link will help in sharing a specific section.

Something like this [link](https://developers.google.com/open-source/gci/resources/getting-started#4_easy_steps)

Links can be added to each and every section or to some specific types of content.

**Are you working on this issue?**
Nope. For GCI 2019

### PR DESCRIPTION
**Pull request by JoshHeng**
Fixes #91

### CODE REVIEW

File: assets/js/shareable-links.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    var url = window.location.href;
+    if (url.search('#') != -1) {
+        url = url.substring(0, url.search('#'));
+    }
+    url += '#' + id;
+    CopyToClipboard(url);
+}
+function AddShareableLinks() {
+    $(":header").mouseenter(function() {
+        $(this).append(`<button class="shareable-link" id="shareable-link" onclick="CopyLink('` + this.id + `')">&#128279;</button>`);
```
[the remark this answers is not in the document]
Comment (junron) [reply assets/js/shareable-links.js]: You can use 
```js
$(this).append(`<button class="shareable-link" id="shareable-link" onclick="CopyLink('${this.id}')">&#128279</button>`);
```
instead, to take advantage of ES6 template strings

[the remark this answers is not in the document]
Comment (JoshHeng) [reply assets/js/shareable-links.js]: Thanks, done

File: assets/js/shareable-links.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    $(":header").mouseenter(function() {
+        $(this).append(`<button class="shareable-link" id="shareable-link" onclick="CopyHeaderLink('${this.id}')">&#128279;</button>`);
+    });
+
+    $(":header").mouseleave(function() {
+        $('#shareable-link').remove();
+    });
+
+    $(".navigation-list-link").mouseenter(function() {
+        $(this).append(`<button class="shareable-link" id="shareable-link">&#128279;</button>`);
```
[the remark this answers is not in the document]
Comment (junron) [reply assets/js/shareable-links.js]: I discovered that you put
```js
/*jshint esversion: 6 */
```
at the top of the file to tell the linter ES6 is supported.
[Source](https://stackoverflow.com/a/27442276/11168593)

**Comment by JoshHeng** [comment]
Note that using the sidebar in the netlify test won't work as the links redirect to the deployed website rather than the test instance

**Comment by YashKumarVerma** [comment]
There is an issue here, both #152 and #155 provide the same functionality, links to sections.

But, since
#152 
- has single commit
- has less files edited
- was opened before
- changes the cursor to a pointer  

whereas in #155 

- has multiple commits
- has more files edited
- was opened later
- changes the cursor to text type
- included jquery as a library

therefore keeping the conventions and good practices in mind, #152 is being merged and #155 is closed its favor.

Thanking @JoshHeng  and @junron  for their contribution.

**Pull request merged**